plugins/woocommerce/src/Blocks/BlockTypes/AbstractBlock.php

```diff
@@ -437,20 +438,21 @@ protected function enqueue_data( array $attributes = [] ) {
 			$this->asset_data_registry->add(
 				'wcBlocksConfig',
 				[
-					'buildPhase'    => Package::feature()->get_flag(),
-					'pluginUrl'     => plugins_url( '/', dirname( __DIR__, 2 ) ),
-					'productCount'  => array_sum( (array) wp_count_posts( 'product' ) ),
-					'restApiRoutes' => [
+					'buildPhase'                => Package::feature()->get_flag(),
```

Comment:
buildPhase will be removed in a follow up PR where we remove the last usage of the flag.